src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -144,6 +144,47 @@
                     return;
                 }
 
+                if (this.selectionListener) {
+                    this.selectionListener();
```

Comment:
name this more specifically, perhaps `removeSelectionListener` ?

Reply:
Good point. Renamed to removeSelectionListener.